docs/using/alerting.md

```diff
@@ -1,44 +1,48 @@
 # Integrated Alerting
 
+*Integrated Alerting* lets you know when certain system events occur.
+
+!!! alert alert-warning "Warning"
+    Integrated alerting is a technical preview and is subject to change.
+
 ---
 
 [TOC]
 
 ---
 
-!!! alert alert-warning "Warning"
-    Integrated alerting is a technical preview and is subject to change.
-
-## Alerts
+## Definitions
 
-An alert has a name, a summary, a description, and a definition as a set of rules.
+- Alerts happen when their criteria (*alert rules*) are met; an *alert* is the result of an *alert rule* firing.
+- Alert rules are based on *alert rule templates*. We provide a default set of templates. You can use them as-is, or modify and extend them to create new ones.
```

Comment:
> You can use them as-is, or modify and extend them to create new ones.

we can't modify our built-in templates, our built-in templates are converted to generated go code which is the part of the managed binary.

the users can modify the templates that they write themselves but not the ones provided by Percona.